lms/templates/course_modes/track_selection.html

```diff
@@ -0,0 +1,207 @@
+        });
+
+        // popover icon toggle
+        $(function(){
+            $('body').click(function (e) {
+                e.stopPropagation();
+                $('.popover').css({"visibility": "hidden", "opacity": 0});
+            });
+            $('.popover-icon').click(function(e){
+                e.stopPropagation();
+                if ($('.popover').css("visibility") == "hidden" || $('.popover').css("visibility") == "" ) {
+                    $('.popover').css({"visibility":"visible", "opacity": 1});   
+                } else {
+                    $('.popover').css({"visibility":"hidden", "opacity": 0});   
+                }
+            });
+        });
+
+        // resize window
+        var onresize = function(e) {
+            width = e.target.outerWidth;
+            if(width < 576) {
+                $('.collapsible-item').css({"display":"none"});
+                $('.collapsible').css({"display":"block"});
+            };
+        }
+        window.addEventListener("resize", onresize);
+
+        // responsive: show more 
+        $(function(){
+            if($(window).width() < 576) {
+            $('.collapsible-item').css({"display":"none"});
+            $('.collapsible').css({"display":"block"});
+            };
+        });
+        $(function(){
+            $('.collapsible').click(function(){
+                $('.collapsible').css({"display":"none"});
+            $('.collapsible-item').css({"display":"list-item"});
+            }); 
+        });
+    </script>
+</%block>
+<%static:webpack entry="Currency">
+    new Currency();
+</%static:webpack>
+
+<%block name="header_extras">
+    <link rel="stylesheet" type="text/css" href="${static.url('paragon/static/paragon.min.css')}" />
+</%block>
+
+<%block name="content">
+    % if error:
+        <div class="wrapper-msg wrapper-msg-error">
+            <div class=" msg msg-error">
+                <span class="msg-icon icon fa fa-exclamation-triangle" aria-hidden="true"></span>
+                <div class="msg-content">
+                    <h3 class="title">${_("Sorry, there was an error when trying to enroll you")}</h3>
+                    <div class="copy">
+                        <p>${error}</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    %endif
+
+    <div id="currency_data" value="${currency_data}"></div>
+
+    <div class="container-md container-fluid mx-auto">
+        <section class="wrapper m-4 m-md-0" dir="ltr"> 
+            <div class="row">
+                <header>
+                    <h3 class="py-3 pt-md-5 pb-md-4">Choose a path for your course in ${course_name}</h3>
+                </header>
+                <form class="form-track-selection" method="post" name="enrollment_mode_form" id="enrollment_mode_form">
+                    <%
+                    b_tag_kwargs = {'b_start': HTML('<b>'), 'b_end': HTML('</b>')}
+                    %>
+                    % if "verified" in modes:
```

Comment:
Technically I don't need this here for now since it's happy-path only, but will need it once the other text is added for non-happy- path.